v1alpha1/proto/grafeas.proto

```diff
@@ -601,6 +601,53 @@ message Discovery {
   Note.Kind analysis_kind = 1;
 }
 
+// In-toto layout
+message Layout {
+  repeated Step steps = 1;
+  google.protobuf.Timestamp expiration_time = 2;
+  string description = 3;
+  repeated Inspect inspection_rules = 4;
+  repeated KeyDetail keys = 5;
+}
+
+// In-toto steps
+message Step {
+  string name = 1;
+  repeated InputOutput expected_materials = 2;
+  repeated InputOutput expected_products = 3;
+  string expected_command = 4;
+  repeated string public_keys = 5;
+  int32 threhold = 6;
+}
+
+message Inspect {
+  string name = 1;
+  repeated InputOutput expected_input = 2;
+  repeated InputOutput expected_output = 3;
+  string run = 4;
+}
+
+// format for materials and products
+message InputOutput {
+  string file_path = 1;
+  // TODO: should this be an enum?
+  string hash_algorithm = 2;
+  string file_hash = 3;
+}
+
+message KeyDetail {
```

Comment:
The format of the layout's `keys` field is defined in [`in_toto.formats. ANY_PUBKEY_DICT_SCHEMA`](https://github.com/in-toto/in-toto/blob/0.2.0/in_toto/formats.py#L45-L47), i.e.:
a dictionary with `keyid`s as dictionary keys and public keys that match either [`securesystemslib.formats.PUBLIC_KEY_SCHEMA`](https://github.com/secure-systems-lab/securesystemslib/blob/v0.10.9/securesystemslib/formats.py#L241-L246) or [`in_toto.gpg.formats.PUBKEY_SCHEMA`](https://github.com/in-toto/in-toto/blob/0.2.0/in_toto/gpg/formats.py#L103-L104) as values.